packages/google_sign_in/google_sign_in_web/example/lib/button_tester.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:google_sign_in_platform_interface/google_sign_in_platform_interface.dart';
+import 'package:google_sign_in_web/google_sign_in_web.dart';
+
+import 'src/button_configuration_column.dart';
+
+// The instance of the plugin is automatically created by Flutter before calling
+// our main code, let's grab it directly from the Platform interface of the plugin.
+GoogleSignInPlugin _plugin =
+    GoogleSignInPlatform.instance as GoogleSignInPlugin;
+
+Future<void> main() async {
+  await _plugin.init();
```

Comment:
Used `await _plugin.initWithParams(const SignInInitParameters(scopes: scopes, clientId: clientId));` here since within `google_sign_in_platform_interface` the `init` function is [stated as deprecated ](https://github.com/flutter/packages/blob/89724994e00637ab5eaaffc4cf71564fbe7fa637/packages/google_sign_in/google_sign_in_platform_interface/lib/google_sign_in_platform_interface.dart#L59)

I guess `init` would work also if the meta tag is added in the `web/index.html` by hand.

Reply:
Also tested a [web only version](https://gist.github.com/c-seeger/5a0d47702e2ff5a3286aa6b045d6c837) that checks scope same as the non web example

Reply:
Yes, this one, since it's a web-only thing, and "idToken"-only thing, I didn't bother with the `scopes` or the `clientId` (which is set in my index.html). I can move this to initWithParams if it's more clear for the example!

Reply:
yes it was more of a side effect of playing around, since the `google_sign_in_web` is part of the `google_sign_in` package and not listed on it's own, I think it's safe to ignore since the more general example should already catch all the cases. :+1: